src/components/DeprecatedBanner/DeprecatedBanner.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import styled from 'styled-components'
+import { ModalConsumer } from '../ModalManager/ModalManager'
+import { theme, Text, SafeLink, Button } from '@aragon/ui'
+import Banner from '../Banner/Banner'
+import Modal from '../Modal/Modal'
+
+const { negative, negativeText, accent, textDimmed } = theme
+
+const DeprecatedBanner = props => (
+  <ModalConsumer>
+    {({ showModal }) => <DeprecatedDao showModal={showModal} {...props} />}
+  </ModalConsumer>
+)
+
+class DeprecatedDao extends React.Component {
+  handleClick = () => {
+    const { dao, showModal } = this.props
+    showModal(DeprecatedModal, { dao })
+  }
+
+  render() {
+    const { children, dao } = this.props
+
+    return (
+      <React.Fragment>
+        <Banner
+          text="This Rinkeby DAO will be deprecated on 1/11/18"
+          textColor={negativeText}
+          onClick={this.handleClick}
+          buttonText="More info"
+          color={negative}
+          dao={dao}
+        />
+        {children}
+      </React.Fragment>
+    )
+  }
+}
+
+const DeprecatedBody = ({ dao }) => (
+  <React.Fragment>
+    <TopParagraph color={textDimmed}>
+      Over the last six months, we’ve made a lot of improvements to aragonOS.
+      Unfortunately a number of those improvements weren’t backwards-upgradeable
+      with the currently deployed contracts that you might have been using on
+      Rinkeby. So we’ve had to make the hard decision to deprecate those old
+      testnet DAOs. Please see{' '}
+      <StyledSafeLink
+        href="https://blog.aragon.one/0-5-dao-deprecation/"
+        target="_blank"
+      >
+        https://blog.aragon.one/0-5-dao-deprecation
+      </StyledSafeLink>{' '}
+      for more details.
+    </TopParagraph>
+    <Text.Paragraph color={textDimmed}>
+      What this means for this DAO is that on 1/11/18 it will no longer be
+      accessible at the current URL. To help manage this migration, and in case
+      you still need access to your old DAOs, we’ll be hosting this DAO created
+      with 0.5 at{' '}
+      <StyledSafeLink href="https://old-app.aragon.org/{dao}">
+        old-app.aragon.org/
+        {dao}
+      </StyledSafeLink>{' '}
+      for a few months after 1/11/18.
```

Comment:
Edits to this copy:

```
On Nov. 1, 2018, this DAO will no longer be accessible on app.aragon.one. To help migrate, and
in case you still need access to this DAO, we will continue hosting this version of Aragon Core
on old-app.aragon.org/#/{dao} until the end of March, 2019.
```

We could use `<time>`s for the dates, and I'd recommend keeping it in text form, to avoid confusion with different orders of parsing (in North America `1/11/2018` would be read as Jan. 11, 2018).

I also made up [March, 2019](https://github.com/aragon/governance/pull/34/files#r222691994) but I think it's better to give a precise date here.